dist/tools/usb-serial/README.md

```diff
@@ -41,29 +41,14 @@ solution):
     # Add serial matching command
     ifneq ($(PROGRAMMER_SERIAL),)
       OOCD_BOARD_FLAGS += -c 'ftdi_serial $(PROGRAMMER_SERIAL)'
-
-      ifeq ($(PORT),)
-        # try to find tty name by serial number, only works on Linux currently.
-        ifeq ($(OS),Linux)
-          PORT := $(firstword $(shell $(RIOTTOOLS)/usb-serial/find-tty.sh "^$(PROGRAMMER_SERIAL)$$"))
-        endif
-      endif
     endif
 
-    # Fallback PORT if no serial was specified or if the specified serial was not found
-    ifeq ($(PORT),)
-        ifeq ($(OS),Linux)
-          PORT := $(firstword $(shell $(RIOTTOOLS)/usb-serial/find-tty.sh))
-        else ifeq ($(OS),Darwin)
-          PORT := $(shell ls -1 /dev/tty.SLAB_USBtoUART* | head -n 1)
-        endif
-    endif
+    PORT_LINUX_EXACT = $(if $(PROGRAMMER_SERIAL),$(firstword $(shell $(RIOTTOOLS)/usb-serial/find-tty.sh "^$(PROGRAMMER_SERIAL)$$")),)
```

Comment:
This changes more than removing `exports`. Only remove the mention to `export` here I would say.

When changing this one, the files using the pattern will also need to be updated, and even put in common which was somehow done in https://github.com/RIOT-OS/RIOT/pull/7695 (with other issues)

```
git grep 'usb-serial/find-tty.sh'
boards/cc2538dk/Makefile.include:PORT_LINUX ?= $(word 2,$(shell $(RIOTTOOLS)/usb-serial/find-tty.sh '^$(PROGRAMMER_SERIAL)'))
boards/mulle/Makefile.include:      PORT := $(firstword $(shell $(RIOTTOOLS)/usb-serial/find-tty.sh '^$(DEBUG_ADAPTER_ID)$$'))
boards/mulle/Makefile.include:      PORT := $(firstword $(shell $(RIOTTOOLS)/usb-serial/find-tty.sh))
boards/pba-d-01-kw2x/Makefile.include:  SERIAL_TTY = $(firstword $(shell $(RIOTTOOLS)/usb-serial/find-tty.sh $(SERIAL)))
dist/tools/usb-serial/README.md:          PORT := $(firstword $(shell $(RIOTTOOLS)/usb-serial/find-tty.sh "^$(PROGRAMMER_SERIAL)$$"))
dist/tools/usb-serial/README.md:          PORT := $(firstword $(shell $(RIOTTOOLS)/usb-serial/find-tty.sh))
makefiles/boards/sam0.inc.mk:  SERIAL_TTY = $(firstword $(shell $(RIOTTOOLS)/usb-serial/find-tty.sh $(SERIAL)))
```

Reply:
Nope, we still keep it in the `serial.inc.mk` for this PR as it only remove `exports` as removing this warning is done by https://github.com/RIOT-OS/RIOT/pull/10342

```
ifeq ($(PORT),)
  $(info Warning: no PORT set!)
endif
```

Otherwise, the whole change should be consistent globally and may require other dependencies to fix boards.